cloudflare_worker/README.md

```diff
@@ -83,6 +75,7 @@ The certificates need to be renewed every 90 days.
    doc](https://docs.digicert.com/manage-certificates/renew-ssltls-certificate/) to renew the certificate.
 1. Overwrite the new-issued `cert.pem` and `issuer.pem` into the folder
    `REPO_ROOT/credentials/`.
+1. Run `cargo run -p config-generator`.
```

Comment:
@antiphoton, sorry for being late to review.  May I suggest that it might be better not to make this an interactive process.  You can do something along the lines of https://github.com/ampproject/amppackager/blob/releases/deploy/gcloud/setup.sh.  The reason I suggest this is that 1) Once the user saves all the relevant input in a file, that file can be version controlled. 2) If the user makes a mistake during an interactive session, that user would have to re-input everything all over again in the second run of that command line utility whereas the approach I mentioned, they could just go back to the shell script and fix the specific problem. Not sure about the urgency of your changes so you can keep this suggestion in mind for future updates.

Reply:
Yes, I agree with this. Rather than setting env variables, we can create a yaml inside `config-generator` folder, and this yaml will be the source-or-truth that generates wrangler config and sxg config. What do you think?

Reply:
SGTM.